docs/data/data-grid/column-dimensions/column-dimensions.md

```diff
@@ -58,6 +58,45 @@ To capture changes in the width of a column there are two callbacks that are cal
 - `onColumnResize`: Called while a column is being resized.
 - `onColumnWidthChange`: Called after the width of a column is changed, but not during resizing.
 
+## Autosizing [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+`DataGridPro` allows to autosize the columns' dimensions based on their content. Autosizing is enabled by default. To turn it off, pass the `disableAutosize` prop to the datagrid.
+
+Autosizing can be used by one of the following methods:
+
+- Adding the `autosizeOnMount` prop,
```

Comment:
Nitpick: We could be more explicit when naming props. For example `autosizeColumnsOnMount`, `disableColumnsAutosize` and `columnsAutosizeOptions`